main.py

```diff
@@ -2,6 +2,36 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
```

Comment:
The genre 'Dramma' seems to be a typo and should be 'Drama'. However, this is corrected later in the code.